buildpacks/python/python/invoker.go

```diff
@@ -20,12 +20,12 @@ type Invoker struct {
 	pythonPath string
 }
 
-func NewInvoker(dependency libpak.BuildpackDependency, cache libpak.DependencyCache) Invoker {
+func NewInvoker(dependency libpak.BuildpackDependency, cache libpak.DependencyCache) *Invoker {
```

Comment:
It would have been nice to supply a mock for the `libpak.DependencyCache` so that we don't need actual testdata on the filesystem (see `testdata/` dir), but it's not possible with the current state of the code, mostly due to the way `libpak.DependencyLayerContributor` is designed -- it takes a concrete type, not an interface.

Still possible to address by a layer of abstraction, but not worth the refactor for this PR